onnxruntime/test/mlas/bench/bench_util.h

```diff
@@ -8,8 +8,12 @@
 #include <functional>
 #include <random>
 
+#include "core/framework/float16.h"
+#include "core/mlas/inc/mlas.h"
+
 template <typename ElementType>
-std::vector<ElementType> RandomVectorUniform(
+typename std::enable_if_t<!std::is_same_v<ElementType, MLAS_FP16>, std::vector<ElementType>>
```

Comment:
nit: would it be simpler to have a specialization for `MLAS_FP16` instead of two `enable_if`s?

Reply:
since this is a .h file included in multiple .cpp files, using specialization will trigger redefinition. so I chose to use enable_if